File: Teledyne DALSA/Sapera/Demos/NET/GrabDemo/VB/GrabDemo.vb
Public Class GrabDemo
   'The main entry point for the application.
   <STAThread()> _
   Shared Sub Main()
      Application.EnableVisualStyles()
      Application.SetCompatibleTextRenderingDefault(False)
      Dim GrabDemo As GrabDemoDlg = New GrabDemoDlg
      If Not GrabDemo.IsDisposed Then
         Application.Run(GrabDemo)
      End If
   End Sub
End Class

File: Teledyne DALSA/Sapera/Demos/NET/GrabDemo/CSharp/GrabDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;


namespace DALSA.SaperaLT.Demos.NET.CSharp.GrabDemo
{
    static class GrabDemo
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            GrabDemoDlg form = new GrabDemoDlg();
            if (!form.IsDisposed)
                Application.Run(form);
        }
    }
}